Description: Use LDFLAGS variable from build environment
Origin: upstream, https://github.com/psi-plus/qconf/commit/851a9d9
Author: Boris Pek <tehnick-8@mail.ru>
Last-Update: 2016-10-20

diff --git a/qconf.pro b/qconf.pro
--- a/qconf.pro
+++ b/qconf.pro
@@ -16,10 +16,19 @@
 }
 
 !isEmpty(CXXFLAGS) {
+	QMAKE_CXXFLAGS = $$CXXFLAGS
 	QMAKE_CXXFLAGS_DEBUG = $$CXXFLAGS
 	QMAKE_CXXFLAGS_RELEASE = $$CXXFLAGS
 }
 
+isEmpty(LDFLAGS) {
+	LDFLAGS=$$(LDFLAGS)
+}
+
+!isEmpty(LDFLAGS) {
+	QMAKE_LFLAGS = $$LDFLAGS
+}
+
 !isEmpty(DATADIR) {
 	contains($$list($$[QT_VERSION]), 4.0.*|4.1.*) {
 		DEFINES += DATADIR=\"$$DATADIR\"
